tests/unittests/CoreGraphics.drawing/ImageComparison.h

```diff
@@ -50,6 +50,8 @@ struct ImageDelta {
     ImageDelta(ImageComparisonResult result, size_t differences, CGImageRef deltaImage)
         : result(result), differences(differences), deltaImage(CGImageRetain(deltaImage)) {
     }
+
+    ImageDelta(ImageDelta&& other) : result(other.result), differences(other.differences), deltaImage(other.deltaImage.release()) {}
```

Comment:
Sorry about that. We could also probably do it by adding a move constructor to unique_cf, but since that's practically dead at this point and no new code should be using it . . . little point eh